spec/models/ability_spec.rb

```diff
@@ -3,9 +3,7 @@
 
 describe 'User' do
   describe 'Abilities' do
-    # automatically becomes admin
-    let!(:first_user) { create(:user) }
-
+    let(:user) { create(:user) }
```

Comment:
Please be careful, you are creating wrong variables. The variable name was first_user and it was (implicitely) an admin.
We already have below a variable user!